app/models/color_scheme.rb

```diff
@@ -572,6 +576,51 @@ def is_dark?
   def is_wcag?
     base_scheme_id == NAMES_TO_ID_MAP["WCAG"] || base_scheme_id == NAMES_TO_ID_MAP["WCAG Dark"]
   end
+
+  def diverge_from_remote
+    new_scheme = dup
+    new_scheme.colors = self.colors_hashes
+    new_scheme.via_wizard = false
+    new_scheme.user_selectable = false
+    new_scheme.base_scheme_id = nil
+    new_scheme.skip_publish = true
+    new_scheme.remote_copy = true
+
+    DistributedMutex.synchronize("color_scheme_fork_#{self.id}") do
+      self.reload
+      if self.base_scheme_id.blank?
```

Comment:
Should we be checking for the present of the record here since `base_scheme_id` may be invalid if there is a bug in the application in the future. By checking for existence, we allow the system to self recover.

Can we also add a test coverage here to ensure multiple calls to this method doesn't end up creating more than 1 remote copy?